eda/src/autogluon/eda/visualization/interaction.py

```diff
@@ -0,0 +1,115 @@
+from abc import ABC
+from typing import Dict, Any, Optional
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from .base import AbstractVisualization
+from .jupyter import JupyterMixin
+from ..state import AnalysisState
+
+__all__ = ["CorrelationVisualization", "CorrelationSignificanceVisualization"]
+
+
+class _AbstractCorrelationChart(AbstractVisualization, JupyterMixin, ABC):
+    def __init__(
+        self,
+        headers: bool = False,
+        namespace: Optional[str] = None,
+        fig_args: Optional[Dict[str, Any]] = None,
+        **kwargs,
+    ) -> None:
+        super().__init__(namespace, **kwargs)
+        self.headers = headers
+        if fig_args is None:
+            fig_args = {}
+        self.fig_args = fig_args
```

Comment:
TIOLI: I'm trying to decide if this is more readable since `fig_args` is only used on next line:

`self.fig_args = fig_args if fig_args is not None else {}`

Reply:
I kept the top since IMO it is easier to read.